README.md

```diff
@@ -121,34 +123,91 @@ export default Total;
 
 ### createSelector([inputSelectors], resultFn)
```

Comment:
Yeah, `...inputSelectors` was added for this new version, we only allow the array in 0.0.2. I'm going to add the `...inputSelectors` signature here too.

BTW, thanks for taking a look at this!